data/downloads_gen.yml

```diff
@@ -4822,6 +4822,212 @@ riak:
               file: riak-2.0.5.tar.gz
               static_file: http://s3.amazonaws.com/downloads.basho.com/riak/2.0/2.0.5/riak-2.0.5.tar.gz
               file_size: 17805547
+  2.1.1:
```

Comment:
Ah crap. I thought that was an artifact of the platform that you'd understand @laurenrother. Working on comparing to upstream and removing what shouldn't bet there.